deye_docker_entrypoint.py

```diff
@@ -20,18 +20,28 @@
 
 from deye_cli import main as cli_main
 from deye_daemon import main as daemon_main
-from deye_config import DeyeConfig
-
-config = DeyeConfig.from_env()
-
-log_format = "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
-
-logging.basicConfig(stream=sys.stdout, format=log_format, level=logging.getLevelName(config.log_level))
-
-log = logging.getLogger('main')
-
-
+from deye_config import DeyeConfig, LOG_DEST_STDOUT, LOG_DEST_STDERR
+
+def setupLogging(config):
+    log_format = "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+
+    supported_log_streams = {
+            LOG_DEST_STDOUT: sys.stdout,
+            LOG_DEST_STDERR: sys.stderr,
+            }
+    if config.log_stream not in supported_log_streams.keys():
+        err = f"Unsupported log stream {config.log_stream}, supported: {','.join(supported_log_streams.keys())}"
+        raise ValueError(err)
```

Comment:
Where is this exception handled? IMO, the process should print an error and exit. No exception traceback is needed.

Reply:
The exception is not handled, but _could_ be handled if that thing was consumed by something different.
This behaves similar as if you would set `LOG_LEVEL` to something unknown/unsupported.

I am however happy to change that, do a print & sys.exit or something.

Reply:
> This behaves similar as if you would set `LOG_LEVEL` to something unknown/unsupported.

Valid point. Then we should wrap call to setupLogging with try...except at line 43 and print a meaningful error message when logger config is incorrect. WDYT?

Reply:
Sounds reasonable to me.
I rebased on the new version of `main` and (I think) implemented what you suggested.
PTAL.